### PR TITLE
Package loga.0.0.5

### PR DESCRIPTION
**Pull request by yutopp**
### `loga.0.0.5`
Logging library for OCaml

---
* Homepage: https://github.com/yutopp/loga
* Source repo: git+https://github.com/yutopp/loga.git
* Bug tracker: https://github.com/yutopp/loga/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/loga/loga.0.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Logging library for OCaml"
+maintainer: ["yutopp@gmail.com"]
+authors: ["yutopp"]
+license: "Boost License Version 1.0"
+homepage: "https://github.com/yutopp/loga"
+bug-reports: "https://github.com/yutopp/loga/issues"
+depends: [
+  "ocaml" {build & >= "4.06.0"}
+  "dune" {build & >= "2.5"}
```
Comment (mseri) [inline packages/loga/loga.0.0.5/opam:10]: The build tags should not be there, for dune this has also to do with forward compatibility issues.

Comment (yutopp) [reply packages/loga/loga.0.0.5/opam:10]: Thank you for your review! I'll fix it.

Comment (mseri) [reply packages/loga/loga.0.0.5/opam:10]: Thanks!

**Comment by yutopp** [comment]
Removed `build` tag from a dependency of `dune`. :pray:

**Comment by mseri** [comment]
@yutopp all the build tags should be removed in this case, since the libraries are needed if you want to compile a project using your library

**Comment by yutopp** [comment]
> all the build tags should be removed in this case

Oops, sorry for my mistakes... :pray: I'll be careful from now on.

> Can you upstream the change

Yes. Thank you for your patch. cc: @kit-ty-kate

**Pull request merged**